.github/workflows/deploy-infra.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy Infra
```

Comment:
not really need to deploy it
just a dhall that can compiles and validate to demonstrate how to use the image is good enough as examples